apps/monitoring/ptlsbox/base/kustomization.yaml

```diff
@@ -4,3 +4,7 @@ namespace: monitoring
 resources:
   - ../../base
   - ../../kube-prometheus-stack/ptlsbox/prometheus-values.enc.yaml
+  - ../../version-reporter
+  - version-reporter.enc.yaml
```

Comment:
ptl was probably for a test, can we follow same principal for these? version-reporter/{ENV}?

Reply:
Oh I forgot to update that. Will sort out now